ugs-platform/ugs-platform-plugin-jog/src/main/java/com/willwinder/ugs/nbp/jog/JogTopComponent.java

```diff
@@ -233,6 +233,14 @@ public void onButtonClicked(JogPanelButtonEnum button) {
                     jogService.setUnits(UnitUtils.Units.MM);
                 }
                 break;
+            case BUTTON_LARGER_STEP:
+                jogService.setStepSizeXY(jogService.getStepSizeXY() * 10);
+                jogService.setStepSizeZ(jogService.getStepSizeZ() * 5);
```

Comment:
Could you switch this to the existing 'jogService.multiplyXYStepSize()`, `jogService.multiplyZStepSize()`? There are also divide methods.